starter-code/index.html

```diff
@@ -3,40 +3,80 @@
 <head>
   <meta charset="UTF-8">
   <title>Spotify</title>
+  <link rel="stylesheet" href="./css/main.css">
 </head>
 <body>
-Premium Discover Help Download
+  <div class="spotify-menu"> 
```

Comment:
Be careful on naming. All your html is wrapped by a container called spotify-menu. Better name it 'main-container/wrapper'